server/routes/journeys/record-review/check-answers/view.njk

```diff
@@ -118,24 +118,21 @@
                     text: 'Contribution'
                 },
                 value: {
-                    text: (attendee.contribution or 'Not provided') | escape | boldAppendStamp | nl2br
+                    html: (attendee.contribution or 'Not provided') | escape | boldAppendStamp | nl2br
```

Comment:
These will follow a direct text edit model rather than an append model won't they? I guess no harm having the boldAppendStamp call in this in case the model gets switched

Reply:
it's the Record a Review CYA screen, so it wouldn't have been appended anyway and didn't need that filter.
we seem to have copy and pasted it on all the 4k char fields tho.